gap/grape.gi

```diff
@@ -91,33 +91,65 @@ function(imm, G, obj, act, adj)
   return D;
 end);
 
-InstallMethod(CayleyDigraph, "for a group with generators",
-[IsGroup, IsHomogeneousList],
-function(G, gens)
-  local adj, D;
-
+InstallMethod(CayleyDigraphCons,
+"for IsMutableDigraph, group, and list of elements",
+[IsMutableDigraph, IsGroup, IsHomogeneousList],
+function(filt, G, gens)
   if not IsFinite(G) then
     ErrorNoReturn("the 1st argument <G> must be a finite group,");
```

Comment:
Here the arguments have the wrong indices if the function is called directly. Of course, this function is undocumented, but this raises the question (which may have already been answered) of whether it should be obscured using `DIGRAPHS_`

Reply:
They also have the wrong indices if `CayleyDigraph` is called with the filter as first argument.